lib/rubocop/cop/rspec/focus.rb

```diff
@@ -26,18 +26,16 @@ class Focus < Cop
           ExampleGroups::GROUPS  +
           ExampleGroups::SKIPPED +
           Examples::EXAMPLES     +
-          Examples::SKIPPED
+          Examples::SKIPPED      +
+          Examples::PENDING
 
         focused = ExampleGroups::FOCUSED + Examples::FOCUSED
 
         FOCUSABLE_SELECTORS = focusable.node_pattern_union
 
-        FOCUS_SYMBOL = s(:sym, :focus)
-        FOCUS_TRUE   = s(:pair, FOCUS_SYMBOL, s(:true))
-
         def_node_matcher :metadata, <<-PATTERN
-          {(send #{RSPEC} #{FOCUSABLE_SELECTORS} ... (hash $...))
-           (send #{RSPEC} #{FOCUSABLE_SELECTORS} $...)}
+          {(send #{RSPEC} #{FOCUSABLE_SELECTORS} <$(sym :focus) ...>)
```

Comment:
This is awesome! 💯 
I thought for some reason this new syntax only works for hashes.